data/players.yaml

```diff
@@ -2897,3 +2897,11 @@
     de:
     - '1165138'
   team: SalzZ
+- name: AlperenAoe
+  country: tr
+  platforms:
+    de:
+    - '874604'
+    - '5453798'
```

Comment:
Can confirm both via Twitch channel

Reply:
874603: Highest Rating 2070 would be currently #198 in 1v1-Ladder
![2021-05-20 17_38_59-AoE2 net — Mozilla Firefox-Screenshot](https://user-images.githubusercontent.com/14062932/119008254-49d52000-b992-11eb-91b3-77ad84bf4649.png)

✔️ Was ever a top 200 player on the primary Age of Empires II competitive ladder. 

(Technically I can't prove it's true, but I think it's ok.)